codec/src/test/java/io/netty/handler/codec/DatagramPacketEncoderTest.java

```diff
@@ -82,11 +82,7 @@ public void testUnmatchedMessageType() {
                 new DefaultAddressedEnvelope<>(1L, recipient, sender);
         assertTrue(channel.writeOutbound(envelope));
         DefaultAddressedEnvelope<Long, InetSocketAddress> output = channel.readOutbound();
-        try {
-            assertSame(envelope, output);
-        } finally {
-            output.release();
-        }
+        assertSame(envelope, output);
```

Comment:
don't we still need to "close" the output ?

Reply:
Not in this case. It's a `DefaultAddressedEnvelope` that just contains a `Long`. Nothing to close.